### PR TITLE
Balance fix

### PR DESCRIPTION
**Pull request by liwenhui-soul**
#### What type of PR is this?
- [ ] bug
- [ ] feature
- [ ] enhancement

#### What problem(s) does this PR solve?
Issue(s) number: 

Description:

#### Checklist:
Tests:
- [ ] Unit test(positive and negative cases)
- [ ] Function test
- [ ] Performance test
- [ ] N/A

Affects:
- [ ] Documentation affected (Please add the label if documentation needs to be modified.)
- [ ] Incompatibility (If it breaks the compatibility, please describe it and add the label.）
- [ ] If it's needed to cherry-pick (If cherry-pick to some branches is required, please label the destination version(s).)
- [ ] Performance impacted: Consumes more CPU/Memory

#### Release notes:

Please confirm whether to be reflected in release notes and how to describe:
> ex. Fixed the bug .....

### CODE REVIEW

File: src/meta/processors/admin/AdminClient.cpp [hunk 1]
```diff
@@ -346,7 +368,7 @@ folly::Future<Status> AdminClient::getResponse(const HostAddr& host,
        respGen = std::move(respGen),
        this]() mutable {
         auto client = clientsMan_->client(host, evb);
-        remoteFunc(client, std::move(req))
+        remoteFunc(host, client, std::move(req))
```
Comment (critical27) [inline src/meta/processors/admin/AdminClient.cpp:371]: If this changes is only for logging, why not just print them in related interfaces?

File: src/meta/processors/job/ZoneBalanceJobExecutor.cpp [hunk 2]
```diff
@@ -305,16 +307,20 @@ Status ZoneBalanceJobExecutor::buildBalancePlan() {
...
+  }
 
   bool emty = std::find_if(existTasks.begin(),
                            existTasks.end(),
                            [](std::pair<const PartitionID, std::vector<BalanceTask>>& p) {
                              return !p.second.empty();
                            }) == existTasks.end();
-  if (emty || rc != nebula::cpp2::ErrorCode::SUCCEEDED) {
+  if (emty) {
+    executorOnFinished_(meta::cpp2::JobStatus::FINISHED);
```
Comment (critical27) [inline src/meta/processors/job/ZoneBalanceJobExecutor.cpp:320]: I can't recall `executorOnFinished_`, what is this for? Why only some of code branch need to call `executorOnFinished_`?

And do we need to handle `LeaderBalanceJobExecutor`?

Comment (liwenhui-soul) [reply src/meta/processors/job/ZoneBalanceJobExecutor.cpp:320]: > LeaderBalanceJobExecutor

finish or stop need to call executorOnFinished_, failed don't need to.

File: src/meta/processors/admin/AdminClient.cpp (hunk 1 above)
Comment (critical27) [reply src/meta/processors/admin/AdminClient.cpp:370]: 👌

**Comment by liwenhui-soul** [comment]
test branch, close now

**Pull request closed without merging**